spec/views/documents/show.html.erb_spec.rb

```diff
@@ -4,6 +4,37 @@
   include TopicsHelper
   before { populate_default_government_bulk_data }
 
+  describe "tags" do
+    let(:tag_field) { build(:tag_field, type: "single_tag", id: "tag_id") }
+    let(:document_type) { build(:document_type, tags: [tag_field]) }
+
+    it "shows the topics when a document has tags" do
```

Comment:
Think this should be  "shows the tags when a document has tags"

Reply:
d'oh! what poor copy and pasting on my behalf. Thanks for catching.